File: ToDoList/app/src/main/java/com/example/todolist/mclass/Task.java
package com.example.todolist.mclass;

import java.util.Calendar;

public class Task {
    private String id;
    private String name;
    private Calendar time;
    private int icon;
    private int rating;
    private String type;

    public Task(String id, String name, Calendar time, int icon, int rating, String type) {
        this.id = id;
        this.name = name;
        this.time = time;
        this.icon = icon;
        this.rating = rating;
        this.type = type;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Calendar getTime() {
        return time;
    }

    public void setTime(Calendar time) {
        this.time = time;
    }

    public int getIcon() {
        return icon;
    }

    public void setIcon(int icon) {
        this.icon = icon;
    }

    public int getRating() {
        return rating;
    }

    public void setRating(int rating) {
        this.rating = rating;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: ToDoListKotlin/app/src/main/java/com/example/todolistkotlin/activity/classes/Task.kt
package com.example.todolistkotlin.activity.classes

import java.util.*

class Task(id: String, name: String, time: Calendar, icon: Int, rating: Int, type: String) {
    private var id: String = ""
    private var name: String = ""
    private var time: Calendar
    private var icon: Int = 0
    private var rating: Int = 0
    private var type: String = ""

    init {
        this.id = id
        this.name = name
        this.time = time
        this.icon = icon
        this.rating = rating
        this.type = type
    }

    fun getId(): String {
        return id
    }

    fun setId(id: String) {
        this.id = id
    }

    fun getName(): String {
        return name
    }

    fun setName(name: String) {
        this.name = name
    }

    fun getTime(): Calendar {
        return time
    }

    fun setTime(time: Calendar) {
        this.time = time
    }

    fun getIcon(): Int {
        return icon
    }

    fun setIcon(icon: Int) {
        this.icon = icon
    }

    fun getRating(): Int {
        return rating
    }

    fun setRating(rating: Int) {
        this.rating = rating
    }

    fun getType(): String {
        return type
    }

    fun setType(type: String) {
        this.type = type
    }
}

